robot/basestation/static/js/app.js

```diff
@@ -552,3 +552,30 @@ $(document).ready(() => {
     }
   })
 })
+
+function pingMCU() {
+  if (millisSince(lastCmdSent) > PING_THROTTLE_TIME) {
+    sendArmRequest('ping', function (msgs) {})
```

Comment:
Im confused because both functions ping arm mcu and ping rover mcu looked identical? What makes this one ping to the arm?

Reply:
```
<button type="button" class="btn fa fa-feed fa-sm btn-secondary" data-toggle="tooltip" title="Ping MCU" onclick="pingMCU()">MCU</button>
```
In Console.html is the only instance using this function. This would mean every page would use the arm request to ping any mcu. So this function needs to be modified by adding a parameter so you can specify to use arm/rover/science request, and it should be accounted for on the front end that the correct parameter is passed to the template (can be done from app.py using template vars I think).

Reply:
Fixed 9f956aecb4dc5143ce719e8f1587c5e72ff30b51